test/CSModule.t.sol

```diff
@@ -2957,6 +3009,400 @@ contract CsmQueueOps is CSMCommon {
     }
 }
 
+contract CsmPriorityQueue is CSMCommon {
+    uint256 constant LOOKUP_DEPTH = 150;
+
+    uint32 constant PRIORITY_QUEUE = 0;
+    uint32 constant MAX_DEPOSITS = 10;
+
+    uint32 REGULAR_QUEUE;
+
+    function setUp() public override {
+        super.setUp();
+        // Just to make sure we configured defaults properly and check things properly.
+        assertNotEq(PRIORITY_QUEUE, csm.QUEUE_LOWEST_PRIORITY());
+        assertNotEq(PRIORITY_QUEUE, csm.QUEUE_LEGACY_PRIORITY());
+        REGULAR_QUEUE = uint32(csm.QUEUE_LOWEST_PRIORITY());
+    }
+
+    function test_enqueueToPriorityQueue_LessThanMaxDeposits() public {
+        uint256 noId = createNodeOperator(0);
+
+        _assertPriorityQueueEmpty();
+        _enablePriorityQueue();
+
+        {
+            vm.expectEmit(true, true, true, true, address(csm));
+            emit IQueueLib.BatchEnqueued(noId, 8);
```

Comment:
I think this event should also contain queueID

Reply:
Done